javelin_sdk/models.py

```diff
@@ -521,7 +521,7 @@ class Config:
 
 class JavelinConfig(BaseModel):
     base_url: str = Field(default="https://api-dev.javelin.live")
-    javelin_api_key: str
```

Comment:
why optional? don't we always need javelin_api_key?

Reply:
wrong commit, trying something, let me revert this line.